aip/0133.md

```diff
@@ -76,6 +76,28 @@ message CreateBookRequest {
   **should not** contain other optional fields except those described in this
   or another AIP.
 
+### Long-running create
+
+Some resources take longer to create a resource than is reasonable for a
+regular API request. In this situation, the API **should** use a long-running
+operation instead:
+
+```proto
+rpc CreateBook(CreateBookRequest) returns (google.longrunning.Operation) {
+  option (google.api.http) = {
+    post: "/v1/{parent=shelves/*}/books"
+  };
+  option (google.longrunning.operation_info) = {
+    response_type: "Book"
+    metadata_type: "OperationMetadata"
+  }
+}
+```
+
+- The response type **must** be set to resource (what the return type would be
```

Comment:
"set to resource" => "set to the resource"?

Reply:
(And elsewhere)

Reply:
Done, thanks.